### PR TITLE
apim multi-region with additional location as copy

### PR DESCRIPTION
**Pull request by solankisamir**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use a parameter for resource locations with the defaultValue set to resourceGroup().location
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values)
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

* Update api version to 2018-01-01
* Use Copy functionality to add regions

### CODE REVIEW

File: 201-api-management-create-with-multiregion/azuredeploy.json [hunk 1]
```diff
@@ -55,10 +46,10 @@
   },
   "resources": [
     {
-      "apiVersion": "2017-03-01",
+      "apiVersion": "2018-01-01",
       "name": "[variables('apiManagementServiceName')]",
       "type": "Microsoft.ApiManagement/service",
-      "location": "[parameters('location')]",
+      "location": "[resourceGroup().location]",
```
Comment (bmoore-msft) [inline 201-api-management-create-with-multiregion/azuredeploy.json]: Add location parameter - see: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#parameters

File: 201-api-management-create-with-multiregion/azuredeploy.parameters.json [hunk 2]
```diff
@@ -8,11 +8,8 @@
         "publisherName": {
             "value": "templateTest"
         },
-        "additionalLocation1": {
-            "value": ""
-        },
-        "additionalLocation2": {
-            "value": ""
+        "additionalLocations": {
+            "value": ["NorthCentralUS", "East US2"]
```
Comment (bmoore-msft) [inline 201-api-management-create-with-multiregion/azuredeploy.parameters.json]: remove this and use the defaultValue for CI

Comment (solankisamir) [reply 201-api-management-create-with-multiregion/azuredeploy.parameters.json]: can you point me to the variable to use here? I could not find any [here](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md#parameters-file-placeholders) 
The requirement here is that `additionLocations` **must** be different from the `resourceGroup` location.

Comment (bmoore-msft) [reply 201-api-management-create-with-multiregion/azuredeploy.parameters.json]: CI will deploy into westus - you already have a defaultValue just use that and remove this from the param file

**Comment by solankisamir** [comment]
#fixed.

**Comment by solankisamir** [comment]
@bmoore-msft can you take a look again?

**Pull request merged**